bedrock/mozorg/templates/mozorg/contribute/tasks/encryption.html

```diff
@@ -23,8 +23,8 @@
     </header>
     <div class="step-content">
       {{ high_res_img('contribute/signup/encryption.png', {'alt': '', 'width': '620', 'class': 'feature-img'}) }}
-      <a href="https://advocacy.mozilla.org/encrypt/signup" class="button-blue" data-action="sign" data-task="encryption" data-step="one" data-complete="true" target="_blank">{{ _('Take the pledge') }}</a>
-      <p>{{ _('<a href="%s" rel="external" target="_blank">Learn more about encryption</a>, why it’s essential and why we need to protect it.')|format('https://advocacy.mozilla.org/encrypt/social/1') }}</p>
+      <a href="https://advocacy.mozilla.org/encrypt/signup" class="button-blue" data-action="sign" data-task="encryption" data-step="one" data-complete="true" target="_blank" rel="noopener noreferrer">{{ _('Take the pledge') }}</a>
+      <p>{{ _('<a href="%s" rel="external" target="_blank" rel="noopener noreferrer">Learn more about encryption</a>, why it’s essential and why we need to protect it.')|format('https://advocacy.mozilla.org/encrypt/social/1') }}</p>
```

Comment:
Note, you also has a duplicate `rel` attribute here, so it would need to be `rel="external noopener noreferrer"`